src/ch05-02-example-structs.md

```diff
@@ -75,79 +76,87 @@ fn area(dimensions: (u32, u32)) -> u32 {
 }
 ```
 
-<span class="caption">Listing 5-9: Specifying the width and height of the
-rectangle with a tuple</span>
+<span class="caption">Listing 5-9: Określenie szerokości i wysokości prostokąta
+przy użyciu krotki</span>
 
-In one way, this program is better. Tuples let us add a bit of structure, and
-we’re now passing just one argument. But in another way, this version is less
-clear: tuples don’t name their elements, so our calculation has become more
-confusing because we have to index into the parts of the tuple.
+Ten program jest, w pewnych aspektach, lepszy.
+Krotki dodają odrobinę organizacji,
+oraz pozwalają nam podać funkcji tylko jeden argument.
+Ale z drugiej strony ta wersja jest mniej czytelna:
+elementy krotki nie mają nazw, a nasze kalkulacje stały się enigmatyczne, 
+bo dany wymiar prostokąta reprezentowany jest przez indeks elementu krotki.
 
-It doesn’t matter if we mix up width and height for the area calculation, but
-if we want to draw the rectangle on the screen, it would matter! We would have
-to keep in mind that `width` is the tuple index `0` and `height` is the tuple
-index `1`. If someone else worked on this code, they would have to figure this
-out and keep it in mind as well. It would be easy to forget or mix up these
-values and cause errors, because we haven’t conveyed the meaning of our data in
-our code.
+Dla kalkulacji pola prostokąta akurat nie ma to znaczenia, ale jeśli chcielibyśmy
+narysować ten prostokąt na ekranie, wtedy już miałoby to znaczenie!
+Musielibyśmy zapamiętać, że szerokość znajduje się w elemencie krotki o indeksie 
+`0`, a wysokość w indeksie `1`.
+Jeśli ktoś inny pracowałby nad tym kodem musiałby rozgryźć to samemu,
+a także to zapamiętać. Omyłkowe pomieszanie tych dwóch wartości nie byłoby zaskakujące,
+a następstwem takiej pomyłki byłyby błędy spowodowane brakiem zawarcia
+kontekstu i znaczenia danych w naszym kodzie.
 
-### Refactoring with Structs: Adding More Meaning
+### Refaktoryzacja ze strukturami: ukazywanie znaczenia
 
-We use structs to add meaning by labeling the data. We can transform the tuple
-we’re using into a data type with a name for the whole as well as names for the
-parts, as shown in Listing 5-10.
+Struktur używamy, aby przekazać znaczenie poprzez etykietowanie danych.
+Możemy przekształcić używaną przez nas krotkę nazywając zarówno
+całość jak i pojedyncze jej części, tak jak w Listingu 5-10.
 
-<span class="filename">Filename: src/main.rs</span>
+<span class="filename">Plik: src/main.rs</span>
 
 ```rust
-struct Rectangle {
+struct Prostokat {
```

Comment:
Tu też bym zostawiła Rectangle (i wszędzie poniżej)

Reply:
jasne, uciekło mi to 😅